docs/collections.md

```diff
@@ -1,107 +1,90 @@
 # 🍱 Collections
 
-## ❓ What are Collections?
+## What are Collections?
 
-[Fetching data is easy in Astro][docs-data]. But what if you wanted to make a paginated blog? What if you wanted an easy way to sort data, or filter data based on part of the URL? Or generate an RSS 2.0 feed? When you need something a little more powerful than simple data fetching, Astro’s Collections API may be what you need.
+Astro Collections help you break up a larger set of data into multiple pages. Examples of use-cases include:
 
-An Astro Collection is similar to the general concept of Collections in static site generators like Jekyll, Hugo, Eleventy, etc. It’s a general way to load an entire data set. But one big difference between Astro Collections and traditional static site generators is: **Astro lets you seamlessly blend remote API data and local files in a JAMstack-friendly way.** To see how, this guide will walk through a few examples. If you’d like, you can reference the [blog example project][example-blog] to see the finished code in context.
+- Pagination: `/posts/1`, `posts/2`, etc.
+- Grouping content by author: `/posts/fred`, `/posts/matthew`, etc.
```

Comment:
```suggestion
- Grouping content by author: `/author/fred`, `/author/matthew`, etc.
```

Minor thing, but it should hint that this can‘t coexist with pagination easily (probably not without some gotchas)